neo/Cryptography/Helper.cs

```diff
@@ -117,7 +117,8 @@ public static byte[] Sha256(this byte[] value, int offset, int count)
         internal static bool Test(this BloomFilter filter, Transaction tx)
         {
             if (filter.Check(tx.Hash.ToArray())) return true;
-            if (filter.Check(tx.Sender.ToArray())) return true;
+            if (tx.Witnesses.Any(p => filter.Check(p.ScriptHash.ToArray())))
```

Comment:
Is better all @erikzhang ?
```suggestion
            if (tx.Witnesses.All(p => filter.Check(p.ScriptHash.ToArray())))
```

Reply:
`Any()` is correct, I think.